src/test/java/com/google/sps/BigBusinessesTest.java

```diff
@@ -0,0 +1,143 @@
+
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import java.util.List;
+import java.util.LinkedList;
+import java.util.ArrayList;
+import java.util.Iterator;
+import static com.google.appengine.api.datastore.FetchOptions.Builder.withLimit;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertSame;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+import static org.hamcrest.CoreMatchers.*;
+
+@RunWith(JUnit4.class)
+public final class BigBusinessesTest {
+  private MapLocation googlesOffice = new MapLocation(40.457177, -79.916696);
+  private List<Listing> sampleListOfBusinesses = new LinkedList<>();
+  private List<Listing> expectedListOfBusinesses = new LinkedList<>();  
+  private Listing[] sampleListingsOfBusinesses = {
+                new Listing("LA Fitness", 
+                  new MapLocation(40.457091, -79.915331), 
+                  3.9, 
+                  null, 
+                  "https://www.lafitness.com/Pages/Default.aspx"),
+                new Listing("west elm", 
+                  new MapLocation(40.456279, -79.915015), 
+                  3.6, 
+                  null, 
+                  "https://www.westelm.com"),
+                new Listing("McDonalds", 
+                  new MapLocation(40.459450, -79.918479), 
+                  2.6, 
+                  null, 
+                  "https://www.mcdonalds.com"),
+                new Listing("East End Brewing Company", 
+                  new MapLocation(40.459391, -79.911782), 
+                  4.7, 
+                  "https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.eastendbrewing.com%2F&psig=AOvVa" + 
```

Comment:
This can be split into more lines using "+".

Reply:
I removed this currently, as the listing class was updated due to the latest changes from Lindsey